app/src/test/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskViewModelTest.kt

```diff
@@ -78,9 +83,9 @@ class AddEditTaskViewModelTest {
     }
 
     @Test
-    fun loadTasks_loading() {
-        // Pause dispatcher so we can verify initial values
-        mainCoroutineRule.pauseDispatcher()
+    fun loadTasks_loading() = runTest {
+        // Override eagerly executing Main dispatcher for just a single test
+        Dispatchers.setMain(StandardTestDispatcher())
```

Comment:
How's this "pausing" the dispatcher?

Reply:
Pausing the Main dispatcher is used to stop it from eagerly executing coroutines, and trigger their execution manually instead. This is what StandardTestDispatcher provides in the new APIs (vs UnconfinedTestDispatcher, which is eager).

(See also: [migration guide section about pausing](https://github.com/Kotlin/kotlinx.coroutines/blob/master/kotlinx-coroutines-test/MIGRATION.md#replace-usages-of-pausedispatcher-and-resumedispatcher-with-a-standardtestdispatcher))